src/Test/WebTestCase.php

```diff
@@ -230,61 +230,42 @@ protected function getContainer(): ContainerInterface
      * to follow the naming format used in $_SERVER.
      * Example: 'HTTP_X_REQUESTED_WITH' instead of 'X-Requested-With'
      *
-     * @param bool|array $authentication
-     * @param array      $params
+     * @param bool  $authentication
+     * @param array $params
      *
      * @return Client
      */
-    protected function makeClient($authentication = false, array $params = []): Client
+    protected function makeClient(bool $authentication = false, array $params = []): Client
```

Comment:
We may rename it to `makeAuthenticatedClient()` because calling `makeClient()` without using authentication is the same as calling `static::createClient(['environment' => $this->environment], $params);`.